e2e/test/iothub/method/MethodE2ETests.cs

```diff
@@ -279,6 +338,27 @@ public static async Task ServiceSendMethodAndVerifyResponseAsync(string deviceId
             serviceClient.Dispose();
         }
 
+        public static async Task<Task> SubscribeAndUnsubscribeMethodAsync(DeviceClient deviceClient, string methodName, MsTestLogger logger)
+        {
+            var methodCallReceived = new TaskCompletionSource<bool>();
+
+            await deviceClient
+                .SetMethodHandlerAsync(
+                methodName,
+                (request, context) =>
+                {
+                    logger.Trace($"{nameof(SubscribeAndUnsubscribeMethodAsync)}: DeviceClient method: {request.Name} {request.ResponseTimeout}.");
```

Comment:
nit: none of this code is required, since this callback is never triggered by the test device

Reply:
We still need to set a callback though. Set -> Unset sequence

Reply:
Yes, you need to set the callback, but it doesn't need any of these log statements (hence the "nit" tag 😄 )